.github/workflows/publish-storybook.yml

```diff
@@ -17,13 +14,16 @@ jobs:
         uses: actions/setup-node@v2
         with:
           node-version: 16.x
-          cache: yarn
 
-      - run: yarn install --frozen-lockfile
+      - run: yarn
```

Comment:
frozen-lockfile is the default when running in CI

Reply:
I'm still a little confused. Because when trying to find the docs online, I found an [open issue](https://github.com/yarnpkg/yarn/issues/4147#issuecomment-321792657) regarding enabling `--frozen-lockfile` by default when in CI.

Reply:
huh, i must be confusing it with pnpm/npm then.

but it doesn't matter, we don't use `--frozen-lockfile` in the other workflows, so no reason to use it here either.

the idea behind it is that CI should fail if the lockfile changes when running install. but it looks like that doesn't even work in yarn v1 (https://github.com/yarnpkg/yarn/issues/5840) so there is no point